test/parallel/test-tls-env-extra-ca.js

```diff
@@ -32,11 +32,11 @@ const server = tls.createServer(options, common.mustCall(function(s) {
   s.end('bye');
   server.close();
 })).listen(0, common.mustCall(function() {
-  const env = Object.assign({}, process.env, {
-    CHILD: 'yes',
-    PORT: this.address().port,
-    NODE_EXTRA_CA_CERTS: fixtures.path('keys', 'ca1-cert.pem')
-  });
+  const env = { ...process.env,
+                CHILD: 'yes',
+                PORT: this.address().port,
+                NODE_EXTRA_CA_CERTS: fixtures.path('keys', 'ca1-cert.pem')
```

Comment:
In general, the change in whitespace here is unnecessary and should be avoided.

Reply:
see 16280f99f2a9e7ccdee12cc3932a222200a9bb18